README.md

```diff
@@ -5,16 +5,23 @@ Application for demonstrating surface blending with ros
 
 Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C3%B6del
 
+### 3rd Party Dependencies
+==============
+- libqd-dev
```

Comment:
+1 on removing the libqd-dev dependency from this list.  Should we include a rosdep step in the instructions?